### ISSUE TITLE
ER: stylelint linter failing after upgrade

### ISSUE THREAD

**Issue by roslynwythe** [hackforla/website#5930]
### Emergent Requirement - Problem
The GitHub Pull Request check "Lint SCSS" is failing.  

### Issue you discovered this emergent requirement in
- many issues including #5900

### Did you have to do something temporarily
- [ ] YES
- [x] NO

### Who was involved
@jaasonw

### What happens if this is not addressed
The GitHub Pull Request check "Lint SCSS" will continue to fail.  Merging is not blocked but the failure creates confusion and developers don't have the benefit of the SCSS linter output.  

### Resources
- #1441
- #2495
- CSS Linter (stylelint): https://stylelint.io/
- GitHub action (Super-Linter): https://github.com/marketplace/actions/super-linter
- [Stylelint configuration](https://github.com/hackforla/website/blob/gh-pages/.stylelintrc.json) 
- [Stylelint Migration Guide 14.0  ](https://stylelint.io/migration-guide/to-14/)

### Recommended Action Items
- [ ] Make a new issue
- [ ] Let a Team Lead know

### Potential solutions [draft]
As stated by @jaasonw in his comment https://github.com/hackforla/website/pull/5900#issuecomment-1811776776, the stylelint configuration file (see resources above) must be updated as described in [Stylelint Migration Guide 14.0  ](https://stylelint.io/migration-guide/to-14/)

**Comment by t-will-gillis** [comment]
Hi Roslyn- 

FYI I had created a previous ER #5048 a few months ago for this issue. I did a lot of exploration with this and I had found the folllowing (see the previous ER also):
- as you note, the failure is occurring because the Superlinter is not recognizing the scss anymore with Stylelint 14, which is the main `customSyntax` error (line 79) that keeps popping up.
<details>
<summary>Current linter error</summary>
<img width="1243" alt="Screenshot 2023-11-24 201139" src="https://github.com/hackforla/website/assets/40799239/fab5c1e0-5c41-4c76-b8fa-97b9eee22496">
</details>

- yes to adding the line regarding  `"extends": ["stylelint-config-standard-scss"]` (I had used "stylelint-config-recommended-scss", but same result)
- After adding the lines above, I had run into a heap of issues. I found adding the "extends" line took care of the `customSyntax` error, i.e. that the Superlinter was recognizing the scss again, however in place of the one error there were 22 new syntax errors. (Here is a workflow run) Errors that seem to be cropping up because our code is using old syntax like "map-get()" (line 77) whereas the updated syntax is  "map.get()".
<details>
<summary>After adding "extends" line</summary>
<img width="1223" alt="Screenshot 2023-11-24 201045" src="https://github.com/hackforla/website/assets/40799239/25c28ca8-9187-4304-8603-e82976bb12f5">
</details>

- Some of these new syntax errors can be automatically addressed by using the [Sass Migrator](https://sass-lang.com/documentation/cli/migrator/), which refactors the Sass to match the latest syntax, 
- However once the Sass Migrator is run, I found that there are still errors, and that Docker either will not load the website at all, or the website runs on Docker with no styling at all.

I realize this is not very helpful...
